src/elfpy/types.py

```diff
@@ -588,3 +592,140 @@ def check_vault_apr(self) -> None:
                 + f"{self.num_trading_days},"
                 + f" not {len(self.vault_apr)}"
             )
+
+
+@dataclass
+class RunSimVariables:
```

Comment:
Let's land as is.  But types.py is gonna get huge.  Let's move these into a simulator/types.py folder/file